applications/asset_tracker/src/gps_controller/gps_controller.c

```diff
@@ -155,7 +155,7 @@ int gps_control_init(struct k_work_q *work_q, gps_event_handler_t handler)
 		return -EINVAL;
 	}
 
-	if (IS_ENABLED(CONFIG_AGPS_SINGLE_CELL_ONLY)) {
+	if (IS_ENABLED(CONFIG_NRF_CLOUD_CELL_POS)) {
```

Comment:
Since cell loc has been separated from GPS/AGPS, I think this `IS_ENABLED` check can be removed.
It can be replaced by something like:
```
if (!IS_ENABLED(CONFIG_NRF9160_GPS) && !IS_ENABLED(CONFIG_GPS_SIM))
``` 
either here, or in asset tracker main, whichever seems best?

Reply:
The code schedule cell_pos as well as agps. Do not want to make significant changes here.